packages/v4/src/content/design-guidelines/usage-and-behavior/modal/modal.md

```diff
@@ -99,9 +99,9 @@ See our [content guidelines](/design-guidelines/content/writing) for additional
 
 | Icon | Use case(s) | Usage |
 | ------------- |-------------|-------------|
-| <i class="fas fa-exclamation-triangle" aria-label="warning"></i> | **Warning:** Caution/ Warning | Use on confirmation dialogs or passive dialogs to indicate a higher level of urgency and importance.|
-| <i class="fas fa-exclamation-circle" aria-label="critical warning"></i> | **Critical Warning:** Information will be deleted/permanent action  | Use on confirmation dialogs or passive dialogs to indicate the highest level of urgency and importance. |
-| <i class="fas fa-info-circle" aria-label="acknowledgement"></i> | **Acknowledgement:** Inform the user of an action or result  | Use on confirmation or passive dialogs to indicate a lower level of urgency. |
+| <span color: rgb(240, 171, 0)> <i class="fas fa-exclamation-triangle" aria-label="warning"></i> </span> | **Warning:** Caution/ Warning | Use on confirmation dialogs or passive dialogs to indicate a higher level of urgency and importance.|
+| <span color: rgb(201, 25, 11)> <i class="fas fa-exclamation-circle" aria-label="critical warning"></i> | **Critical Warning:** Information will be deleted/permanent action  | Use on confirmation dialogs or passive dialogs to indicate the highest level of urgency and importance. |
+| <span color: rgb(43, 154, 243)> <i class="fas fa-info-circle" aria-label="acknowledgement"></i> | **Acknowledgement:** Inform the user of an action or result  | Use on confirmation or passive dialogs to indicate a lower level of urgency. |
```

Comment:
Need a closing </span> tag here too - So it should be:
`<span color: rgb(43, 154, 243)> <i class="fas fa-info-circle" aria-label="acknowledgement"></i> </span>`